### ISSUE TITLE
Cannot run GraphiQL query from document with multiple operations

### ISSUE THREAD

**Issue by dylanscott** [apollographql/apollo-client-devtools#129]
It seems like GraphiQL is supposed to support running a single query from a document with multiple operations, based on the fact that it exposes this dropdown on the "run" button:

![screen shot 2018-06-19 at 9 26 48 am](https://user-images.githubusercontent.com/491393/41610819-112b1972-73a3-11e8-8b1a-653b27527807.png)

But when I click one of the queries in the dropdown, I get this error:

![screen shot 2018-06-19 at 9 26 55 am](https://user-images.githubusercontent.com/491393/41610840-1d513bc8-73a3-11e8-98b7-094436ce4b2b.png)

This error appears to be happening on the client side. We never send a network request for the query at all.

**Comment by justinanastos** [comment]
I just joined the devtools project and I'm going through all the open issues.

Thanks for the bug report @dylanscott . I confirmed this is still an issue with the latest devtools. We'll look into it!

**Comment by dylanscott** [comment]
Thanks @justinanastos!

**Comment by mjlyons** [comment]
I have a rough fix here: https://github.com/apollographql/apollo-client-devtools/commit/d5e1205adaf71ba3a1ae316f3cca915e7cb84652.

Unfortunately I screwed up sending my PR for #108 (PR #204) and put it on my fork's master. I think I can't send my commit above as a PR until PR #108 is pulled in (or I burn my fork).

There might be a cleaner way to do this change, so happy to take any code review feedback in the meantime.

Thanks,
Mike

**Comment by fletc3her** [comment]
It doesn't look like this was ever closed.

I'm encountering something similar. I was testing a query operation fine but as soon as I added a mutation I started getting an error "Error while executing GraphQL statement Error: Could not get operation from the document" even though the pop-up suggested I would be running only one of the operations. 

Manually deleting one of the operations allows the explorer to execute the other operation.
